corona-warn-app/cwa-app-android
# Top margin certificate details (EXPOSUREAPP-13101)

How to test:
- Scan invalid certificate (e.g.  cwa vc gen --exp 2021-07-22T08:57:25Z -e mock-cloud)
- Certificate detail view has top margin

Before:

![Screenshot_20220516-111633_CWA TEST](https://user-images.githubusercontent.com/22813752/168562939-68d5b5c9-72b0-4f99-b511-fe7147583b84.jpg)

After:

![Screenshot_20220516-111344_CWA TEST](https://user-images.githubusercontent.com/22813752/168562973-4ad021bd-d1b0-4fa7-a01d-7f96802b150b.jpg)


In Ticket [EXPOSUREAPP-11516](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11516) padding was removed from the card.

This works as long the certificate is valid and the covpass info title and button is shown, since the padding is applied

to the info title.

When the certificate is invalid, covpass info title and button is not shown, so padding has to be applied
